en-US/firefox/enterprise/index.lang

```diff
@@ -0,0 +1,159 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/enterprise/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/enterprise/
+
+
+# HTML page title
+;Get Firefox for your enterprise with ESR and Rapid Release
+Get Firefox for your enterprise with ESR and Rapid Release
+
+
+# HTML page description
+;Get unmatched data protection on the release cadence that suits you with Firefox for enterprise. Download ESR and Rapid Release.
+Get unmatched data protection on the release cadence that suits you with Firefox for enterprise. Download ESR and Rapid Release.
+
+
+;Enterprise
+Enterprise
+
+
+;Overview
+Overview
+
+
+;Downloads
+Downloads
+
+
+;Resources
+Resources
+
+
+;Get Firefox for your enterprise
+Get Firefox for your enterprise
+
+
+;Get the <a href="%s">Firefox Extended Support Release or Rapid Release</a> browser for comprehensive data security and data protection.
+Get the <a href="%s">Firefox Extended Support Release or Rapid Release</a> browser for comprehensive data security and data protection.
+
+
+# CTA button
+;Download
+Download
+
+
+;Firefox Privacy Notice
+Firefox Privacy Notice
+
+
+;Unmatched data protection — on the release cadence that suits you
+Unmatched data protection — on the release cadence that suits you
+
+
+;The Firefox browser is open source, provides Enhanced Tracking Protection and soon will support DNS over HTTPS — all part of our longstanding commitment to data protection.
+The Firefox browser is open source, provides Enhanced Tracking Protection and soon will support DNS over HTTPS — all part of our longstanding commitment to data protection.
+
+
+;The Firefox browser is open source and provides Enhanced Tracking Protection — all part of our longstanding commitment to data protection.
+The Firefox browser is open source and provides Enhanced Tracking Protection — all part of our longstanding commitment to data protection.
+
+
+# your data remains yours - it’s your business and no one else’s
+;Your data stays your business
```

Comment:
it's not missing a word, the comment above was explaining the meaning of the sentence since it's a colloquialism. It makes more sense if read "your data", "stays your business". 

I had the exact same question when I was shown the copy and they had to clarify it for me too :)